html/modules/custom/hr_paragraphs/hr_paragraphs.module

```diff
@@ -938,11 +938,42 @@ function hr_paragraphs_library_info_alter(&$libraries, $extension) {
  * Implements hook_menu_local_tasks_alter().
  */
 function hr_paragraphs_menu_local_tasks_alter(&$data, $route_name) {
+  $weights = [
+    'hr_paragraphs.operation' => -10,
+    'group.view' => 0,
+    'group.edit_form' => 5,
+    'group.delete_form' => 10,
+    'group.version_history' => 15,
+    'views_view:view.group_members.page_1' => 20,
+    'views_view:view.subgroups_of_a_group.page_1' => 30,
+    'views_view:view.group_nodes.page_1' => 40,
+    'content_translation.local_tasks:entity.group.content_translation_overview' => 100,
+    'hr_paragraphs.operation.operations' => 999,
+    'group.content' => 999,
+  ];
+
+  // Re-order tabs.
+  foreach ($data['tabs'][0] as $key => &$tab) {
```

Comment:
Sometimes this gives `
Error message
Warning: foreach() argument must be of type array|object, null given in hr_paragraphs_menu_local_tasks_alter() (line 956 of modules/custom/hr_paragraphs/hr_paragraphs.module). ` 

e.g. on https://operations.test/group/13/content/create/subgroup%3Acluster - but refreshing the page clears it. Should probably add a check for null

Reply:
Fixed in